src/config/section/network.js

```diff
@@ -47,7 +47,7 @@ export default {
       }, {
         name: 'Egress Rules',
         component: () => import('@/views/network/EgressConfigure.vue'),
-        show: record => record.service && record.service.filter(x => x.name === 'Firewall').length > 0
+        show: () => true
```

Comment:
@rhtyd This line was the source of the issue with refreshing the page and the tab changing away from Egress. It's linked with the fact that there needs to be an API call before getting this info, so the tab is always hidden by default (whilst waiting for the data) so that when you click the refresh button, the tab isn't there, therefore making it unable to maintain that active tab state.
So the potential solution I've implemented is showing the tab at all times, and just displaying the default "No Data" message when there are no records that match.